runHarness/Workload/Workload.pm

```diff
@@ -811,12 +842,20 @@ sub writeUsersTxt {
 	
 	foreach my $appInstance (@$appInstancesRef) {
 		my $instanceNum = $appInstance->instanceNum;
-		my $users = $appInstance->getUsers();
-		my $outString      = "Workload $workloadNum, App Instance $instanceNum: $users Users";
-		$console_logger->info($outString);
-		print $fileOut "$outString\n";
+		my $loadPathType = $appInstance->getParamValue('loadPathType');
+		if ($loadPathType eq 'fixed') {
+			my $users = $appInstance->users;
+			if ($users) {
+				my $outString      = "Workload $workloadNum, App Instance $instanceNum: $users Users";
+				$console_logger->info($outString);
+				print $fileOut "$outString\n";
+			}
+		} elsif ($loadPathType eq 'findMax') {
+				my $outString      = "Workload $workloadNum, App Instance $instanceNum: Single-Run FindMax";
```

Comment:
In theory, FindMax can also be used for non Single-Run runs?

Reply:
What I was trying to convey is that for this run, that workload/appInstance will be running with a varying load.  I will remove the Single-Run so that it just says FindMax.